Gopkg.lock

```diff
@@ -3,595 +3,481 @@
 
 [[projects]]
```

Comment:
I have no idea why all the changes to this file has happened, but I would prefer if they are reverted ;) We will be moving to go modules #1070  as soon as we move to 1.13, so even if this is because dep changed we prefer if we don't have unnecessary amount of changes ...